docs/reference/operators/otel-metrics.md

```diff
@@ -2,8 +2,38 @@
 title: otel-metrics
 ---
 
-TODO
+The otel-metrics operator handles collecting and exporting metrics using the
+[Prometheus exporter](https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/prometheus/). When setting
+`otel-metrics-listen=true`, it will serve http requests on "0.0.0.0:2224" (configurable using
+`otel-metrics-listen-address`) with the metrics available at "/metrics".
 
-## Parameters
+## Global Parameters
 
-TODO
+### `otel-metrics-listen`
+
+Enables the Prometheus exporter on the address given by `otel-metrics-listen-address` if set to `true`.
+
+Default: `false`
+
+### `otel-metrics-listen-address`
+
+The listen address that should be serving Prometheus requests.
+
+Default: `0.0.0.0:2224`
+
+## Instance Parameters
+
+### `otel-metrics-name`
+
+Overrides the name of a datasource and explicitly sets it as name for the export. This is mandatory if you
+want to export the metrics. Use a name that is unique to the gadget+params combination to avoid collision of metrics.
+
+Default: ``
+
+### `otel-metrics-print-interval`
+
+Interval in which metrics should be emitted as human-readable text. This only has effect for data sources that are
```

Comment:
Why we started talking about human-readable? Is there other kid of output, is it configurable?

Reply:
This was added to mimic the behavior of profile gadgets without having full rendering support of metrics to the CLI in a better way (I think it only makes sense with a proper UI). So this converts metrics to the histogram output that we've shown in the built-in versions and annotate it as "text" so the CLI knows to just print it (and not consider it as columns or such.)
You can also access metrics using JSON.